configure.ac

```diff
@@ -0,0 +1,53 @@
+dnl Process this file with autoconf to produce configure
+
+# Copyright 2017 Endless Mobile, Inc.
+
+dnl ------------
+dnl configure.ac
+dnl ------------
+dnl Please keep this file well-commented. Autotools involve a lot of magical
+dnl incantations, and it is easy to mess things up if you don't know what you
+dnl are doing.
+
+# Initialization
+# --------------
+# Initialize Autoconf: package name, version, bug report address, tarball name,
+# website
+AC_INIT([Basin], [0],
+    [], [basin], [http://endlessm.com])
+# Initialize Automake: enable all warnings and do not insist on GNU standards
+# no-portability suppresses warnings about syntax specific to GNU make
+AM_INIT_AUTOMAKE([-Wall -Wno-portability foreign 1.11 tar-ustar])
+# Avoid spewing garbage over the terminal ('make V=1' to see the garbage)
+AM_SILENT_RULES([yes])
+# Keep Autotools macros local to this source tree
+AC_CONFIG_MACRO_DIR([m4])
+
+AC_CACHE_SAVE
+
+# Required build tools
+# --------------------
+# Make sure we can create directory hierarchies
+AC_PROG_MKDIR_P
+# Various tools
+AC_PROG_AWK
+
+AC_CACHE_SAVE
+
+# Required libraries
+# ------------------
+EOS_CHECK_GJS_GIR([Gio], [2.0])
+EOS_CHECK_GJS_GIR([GLib], [2.0])
+EOS_CHECK_GJS_GIR([EosShard], [0])
+EOS_CHECK_GJS_GIR([Xapian], [1.0])
+
+AC_CACHE_SAVE
```

Comment:
I would also recommend using `AX_CHECK_GIR_SYMBOLS_GJS` to make sure `Xapian.Document.add_numeric_value` is present.